modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-proxy-protocol_{context}"]
+= Configuring PROXY protocol for an Ingress Controller
+As a cluster administrator, you can use PROXY protocol to enable communication between the Ingress Controller and an external load balancer. An Ingress Controller is typically published with a load balancer. The connections that the Ingress Controller receives are from the load balancer, so the source address has the load balancer address, instead of the original connection address. The PROXY protocol enables a load balancer to preserve the original source addresses and include them when forwarding connections to an Ingress Controller, which can then use the original source addresses for logging.
```

Comment:
Maybe "enhance communication" would be more accurate:  PROXY protocol isn't required for the load balancer to forward traffic to the Ingress Controller, but PROXY protocol adds additional information (as your second sentence describes).  
```suggestion
As a cluster administrator, you can use PROXY protocol to enhance communication between the Ingress Controller and an external load balancer. An Ingress Controller is typically published with a load balancer. The connections that the Ingress Controller receives are from the load balancer, so the source address has the load balancer address, instead of the original connection address. The PROXY protocol enables a load balancer to preserve the original source addresses and include them when forwarding connections to an Ingress Controller, which can then use the original source addresses for logging.
```